webpack/assets/javascripts/pages/statistics_page.js

```diff
@@ -0,0 +1,48 @@
+import donut from '../services/c3DonutChartService';
+
+export default {
+    generateChart: generateChart,
+    generateModalChart: generateModalChart
+};
+
+function generateChart(details, data) {
+    let dataEventHandlers = {
+        onclick: getClickHandler(details.search)
+    };
+
+    let config = donut.getDonutConfig(data, '#' + details.id + 'Chart', dataEventHandlers);
+
+    donut.generate(config);
+
+    function getClickHandler(url) {
+        if (url) {
+            // eslint-disable-next-line no-unused-vars
+            return function (d, i) {
```

Comment:
the c3 callback has 2 parameters: the data clicked and the element clicked. code changed to more meaningful names. I like to list all parameters in 3rd party callbacks (including unused)  as a sort of documentation